### ISSUE TITLE
Refresh GitLab project access token

### ISSUE THREAD

**Issue by josephdt12** [GoogleCloudPlatform/cloud-foundation-toolkit#2168]
GitLab project access tokens have a required maximum expiration date of 1 year.
Need to update the GitLab secret at infra/terraform/test-org/ci-triggers/secrets.tf with the refreshed token.

**Comment by bharathkkb** [comment]
@josephdt12 is this just for tracking as a reminder?

**Comment by josephdt12** [comment]
Yeah, the tokens only last a year unfortunately. Perhaps we can have something do this automatically, but I haven't investigated that yet.
